CONTRIBUTING.md

```diff
@@ -222,6 +222,21 @@ Situations when it may be fine to ignore a MiMa issued warning include:
 
 The binary compatibility of the current changes can be checked by running `sbt +mimaReportBinaryIssues`.
 
+## Wire compatibility
+
+Changes to the binary protocol of remoting, cluster and the cluster tools require great care so that it is possible
+to do rolling up and downgrades. 
```

Comment:
I think we shouldn't give the impression we support (rolling) downgrades - I can't think of any cases offhand, but it feels harder to guarantee.
```suggestion
to do rolling upgrades.
```

Reply:
This aspect is specifically why I wanted to add this new section. 

There is a very thin line between rolling downgrades and keeping old nodes being able to communicate with new ones during the rolling upgrade. But maybe that could be phrased differently - I don't see this as what we promise though, more what we aim for when doing changes.

Reply:
:+1: to explicitly just saying rolling upgrades which includes old nodes being able to communicate with new nodes during the process